sites/main-site/src/content/events/2025/hackathon-march-2025/montreal-mcgill.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: 'Hackathon - March 2025 (Montreal McGill)'
+subtitle: 'Local node of the nf-core hackathon in Montreal at McGill offices'
+shortTitle: '🇨🇦 Montreal McGill'
+type: 'hackathon'
+startDate: '2025-03-24'
+startTime: '09:00+01:00'
+endDate: '2025-03-26'
+endTime: '17:00+01:00'
+locations:
+    - name: Montreal McGill offices
+      address: |
+          Lovelace Conference Room, Suite 1800, 1010 Rue Sherbrooke Ouest, Montreal, QC H3A 2R7
+      links:
+          - https://www.mcgill.ca/facilities/downtown-campus
+      geoCoordinates: [45.502250, -73.575490]
+layout: '@layouts/events/HackathonMarch2025.astro'
+---
+
+# Contacts
+
+Main contact: Florian Wuennemann ([flowuenne@gmail.com](mailto:flowuenne@gmail.com))
+[<i class="fab fa-slack"></i> Florian Wuennemann](https://nfcore.slack.com/team/UU10KMQ1F) ([flowuenne@gmail.com](mailto:flowuenne@gmail.com))
+
+# Participation
+
+Anybody who is interested **can** attend. Please make sure to select the local site during registration and please contact Florian Wuennemann prior to the event since we are limited to approximately
```

Comment:
```suggestion
Anybody who is interested **can** attend. 
Please make sure to select the local site during registration and please contact Florian Wuennemann prior to the event since we are limited to approximately
```